src/jit/emit.cpp

```diff
@@ -2953,6 +2953,36 @@ void                emitter::emitDispVarSet()
 
 /*****************************************************************************/
 #endif//DEBUG
+
+//------------------------------------------------------------------------  
+// emitSetSecondRetRegGCType: Sets the GC type of the second return register for instrDescCGCA struct.  
+//  
+// Arguments:  
+//    id            - The large call instr descriptor to set the second GC return register type on.  
+//    secondRetSize - The EA_SIZE for second return register type.  
+//  
+// Return Value:  
+//    None  
+//  
+
+void            emitter::emitSetSecondRetRegGCType(instrDescCGCA* id, emitAttr secondRetSize)
+{
+#if defined(FEATURE_MULTIREG_RET)
```

Comment:
As per our offline discussion. The FEATURE_MULTIREG_RET should not be used for this feature. Only Amd64 Unix can return struct in 2 (or more) registers that contain GC refs, so use FEATURE_UNIX_AMD64_STRUCT_PASSING define instead (as it was in the first version of this PR).
Note that this also removed the codegenlegacy.cpp and all ARM related from this change set.